networkit/plot.py

```diff
@@ -23,35 +23,62 @@
 else:
 	have_seaborn = True
 
-def nodeProperty(data, label, sorted=True, yscale="linear", xscale="linear"):
+def nodeAttributes(G, attribute1=None, attribute2=None):
```

Comment:
Better use one parameter and check whether it is of type `nk.graph.NodeAttribute` or `tuple(nk.graph.NodeAttribute)`. The documentation should also reflect this: a) name appropriate datatypes in parameter list and b) shortly describe behavior in function description (What happens if one attribute is passed? What happens if two are passed?).

Therefore mention, that